src/blocks/input.css

```diff
@@ -0,0 +1,65 @@
+.input {
+  background-image: url(../images/Search.svg);
+  background-color: #fff;
+  background-repeat: no-repeat;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  flex-grow: 1;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: Avenir, serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.input::placeholder {
+  color: #3d4e61;
+}
+
+.input:hover {
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.input:focus {
+  background-repeat: no-repeat;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: bold;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  outline: none;
+}
+
+.input:focus::placeholder {
+  opacity: 0;
+}
+
+.input--big {
+  font-size: 16px;
+  line-height: 22px;
+  background-size: 19px;
+  background-position: 24.5px 24px;
+  padding: 23px 0 23px 62px;
```

Comment:
height should be 70px
![image](https://github.com/user-attachments/assets/1f488fd5-17e6-47cd-a885-c13b9b02dcfd)